test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
Oh, gotcha. Yeah, was not reading this correctly.

It would be nice to be able to specify the ruleset somehow when calling `axeCheck`, so we could catch regressions in updated apps before everything is switched over.